mozetl/topline/topline_summary.py

```diff
@@ -214,7 +215,7 @@ def transform(dataframe, start, mode):
         .join(searches, attributes, "outer")
         .join(hours, attributes, "outer")
         .withColumnRenamed('country', 'geo')
-        .withColumn('crashes', F.lit(0L))
+        .withColumn('crashes', F.lit(0))
```

Comment:
please double-check that this creates a column of type long in both python 2 and python 3

Reply:
This was a really good catch, I added schema validation in a test.